test/gurka/test_avoids.cc

```diff
@@ -214,6 +217,50 @@ TEST_P(AvoidTest, TestAvoid2Polygons) {
   };
 }
 
+TEST_P(AvoidTest, TestAvoidShortcuts) {
+  auto node_b = avoid_map.nodes.at("B");
+  auto node_c = avoid_map.nodes.at("C");
+  auto node_e = avoid_map.nodes.at("E");
+  auto dx = std::abs(node_c.lng() - node_b.lng());
+  auto dy = std::abs(node_b.lat() - node_e.lat());
+
+  // create a small polygon on the shortcut: should fail if shortcuts are also avoided
+  //                x-x
+  //      A------B--|-|-C--F--G
+  //                x-x
```

Comment:
you can literally make those x's into osm nodes and use their lat lons directly from gurka to make the polygon rather than relying on hand done math below.

```suggestion
  //                h-i
  //      A------B--|-|-C--F--G
  //                j-k
```

Reply:
made the whole test suite so much more legible!